### PR TITLE
feature: support 004 accounts

### PR DESCRIPTION
**Pull request by baptiste-grob**
- The script is now using Typescript + dependencies to ensure compatibility with SNJS and avoiding hard-coding as many scripts as possible
- At build time, the generated script is inlined in the `decrypt.html` file so users still only have to download one file

### CODE REVIEW

**Review by mobitar — CHANGES_REQUESTED** [review]
Two things:

1. Should use storage policy ephemeral as not to persist any data: https://github.com/standardnotes/snjs/blob/004/lib/services/storage_service.ts#L105

2. Should call application.signOut after use to destroy any data that may otherwise have been saved anywhere.

**Comment by baptiste-grob** [comment]
setPersistencePolicy is only called upon signing in or registering. Is it safe to set it at any time during the app lifecycle?

**Comment by mobitar** [comment]
Yup should be.

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+    }
+    return keychain[identifier];
+  }
+
+  async setNamespacedKeychainValue(value: any, identifier: any) {
+    let keychain = await this.getRawKeychainValue();
+    if (!keychain) {
+      keychain = {};
+    }
+    localStorage.setItem(
```
Comment (mobitar) [inline src/index.ts:116]: This device interface really should be memory based overall. It's another insurance policy against data persistence. So the keychain should be an in-memory data structure, as well as the payload store. Really we shouldn't see any references of localStorage.

**Review by mobitar — APPROVED** [review]

**Pull request merged**